backend/src/test/java/heartbeat/service/report/GenerateReporterServiceTest.java

```diff
@@ -812,6 +829,23 @@ void shouldGenerateForBoardCsvWhenCallGenerateReporterWithBoardMetric() throws I
 		Files.deleteIfExists(mockBoardCsvPath);
 	}
 
+	@Test
+	public void shouldDeleteExpireCsvWhenExportCsvWithCsvOutsideThirtyMinutes() throws IOException {
+		Files.createFile(mockPipelineCsvPath);
+
+		generateReporterService.deleteExpireCSV(System.currentTimeMillis());
+
+		boolean isFileDeleted = Files.notExists(mockPipelineCsvPath);
+		Assertions.assertTrue(isFileDeleted);
+	}
+
+	@Test
+	public void shouldThrowExceptionWhenDeleteCSV() {
+		GenerateReporterService reporterService = Mockito.spy(generateReporterService);
+		doThrow(new RuntimeException()).when(reporterService).deleteOldCSV(anyLong());
```

Comment:
Where is the assertion ????
Serious about writing the test case?
maybe you can pair it with a Java master/guru